docs/FAQ.md

```diff
@@ -0,0 +1,177 @@
+# FAQ
+
+## Frequently Asked Questions
+
+```{toctree}
+:maxdepth: 3
+
+https://github.com/reactive-firewall/multicast/.github/CODE_OF_CONDUCT.md
+https://github.com/reactive-firewall/multicast/.github/CONTRIBUTING.md
+```
+
+### How do I get this running?
+
+(assuming python3 is setup and installed)
+
+```bash
+# cd /MY-AWSOME-DEV-PATH
+git clone https://github.com/reactive-firewall/multicast.git multicast
+cd ./multicast
+git checkout stable
+# make clean ; make test ; make clean ;
+make install ;
+python3 -m multicast --help ;
+```
+#### DONE
+
+If all went well `multicast` is now installed and working :tada:
+
+
+### How do I use this to receive some UDP Multicast?
+
+(assuming project is setup and installed and you want to listen on 0.0.0.0)
+
+```bash
+# cd /MY-AWSOME-DEV-PATH
+python3 -m multicast HEAR --use-std --port 59595 --join-mcast-groups 224.0.0.1 --bind-group 224.0.0.1
+```
+
+Caveat: RCEV is much more usefull if actually used in a loop like:
+
+```bash
+# cd /MY-AWSOME-DEV-PATH
+while true ; do # unitl user ctl+c inturupts
+python3 -m multicast RECV --use-std --port 59595 --join-mcast-groups 224.0.0.1 --bind-group 224.0.0.1
+done
+```
+
+
+### How do I use this to send UDP Multicast?
+
+(assuming project is setup and installed)
+
+```bash
+# cd /MY-AWSOME-DEV-PATH
+python3 -m multicast SAY --mcast-group 224.1.1.2 --port 59595 --message "Hello World!"
+```
+
+
+### What is the basic API via python (instead of bash like above):
+
+#### Caveat: this module is still a BETA
+[Here is how it is tested right now](https://github.com/reactive-firewall/multicast/blob/cdd577549c0bf7c2bcf85d1b857c86135778a9ed/tests/test_usage.py#L251-L554)
+
+```python3
+import multicast as multicast
+from multiprocessing import Process as Process
+
+# setup some stuff
+_fixture_PORT_arg = int(59595)
+_fixture_mcast_GRP_arg = """224.0.0.1"""  # only use dotted notation for multicast group addresses
+_fixture_host_BIND_arg
+_fixture_HEAR_args = [
+	"""--port""", _fixture_PORT_arg,
+	"""--join-mcast-groups""", _fixture_mcast_GRP_arg,
+	"""--bind-group""", _fixture_mcast_GRP_arg"
+]
+
+# spwan a listening proc
+
+def inputHandle()
+	buffer_string = str("""""")
+	buffer_string += multicast.recv.hearstep([_fixture_mcast_GRP_arg], _fixture_PORT_arg, _fixture_host_BIND_arg, _fixture_mcast_GRP_arg)
+	return buffer_string
```

Comment:
assume Tabs instead of spaces (as is consistent for the rest of the project)
```suggestion
	test_RCEV = multicast.recv.McastRECV()
	buffer_string = str("""""")
	buffer_string += test_RCEV._hearstep([_fixture_mcast_GRP_arg], _fixture_PORT_arg, _fixture_host_BIND_arg, _fixture_mcast_GRP_arg)
	return buffer_string
```